### ISSUE TITLE
Missing support for DEFAULT color ?

### ISSUE THREAD

**Issue by sebastien-f** [L-Blondy/tw-colors#7]
I defined a whole new color with 
```js
const colors = {
    "english-violet": {
        "DEFAULT": "#514663",
        "100": "#ffffff",
        "99": "#eeedef",
        "95": "#dcdae0",
        "90": "#a8a3b1",
        "80": "#857e92",
        "70": "#625973",
        "60": "#514663",
        "50": "#493f59",
        "40": "#41384f",
        "30": "#393145",
        "20": "#312a3b",
        "10": "#18151e",
        "0": "#000"
    }
};
```

Then added it to the theme:
```js
  plugins: [
    createThemes({
                   light: {
                      'primary': colors["english-violet"],
                      'secondary': 'darkblue',
                      'brand': '#F3F3F3',
                   }
                })
  ],
```

If I'm using `bg-primary` with the theme applied, I don't get any color. Other variations works : 

```html
            <div class="bg-primary">
                Doesn't work
            </div>
            <div class="bg-primary-60">
                Works as intended
            </div>
            <div class="bg-primary-DEFAULT">
                Works also
            </div>

```

Without using the theme, the following also [work as intended](https://tailwindcss.com/docs/customizing-colors#color-object-syntax) : 

```html
            <div class="bg-english-violet">
                Works as intended
            </div>
```

**Comment by sebastien-f** [comment]
That's really weird, I've been doing more testing and adding more colors to the theme, and some of them works with DEFAULT, some don't. 
I tried to remove the hyphen in the name (just in case), no change. 
Also tried to use dot notation rather than brackets (I can't see why that would do something, but as it is weird...), no change either.
For what it's worth, VS Code intellisense with official tailwind extension sees the default colors for those which works, and only sees the *-DEFAULT for those that doesn't work.
Example of one that works and properly see the DEFAULT : 

```js
    "error": {
        "DEFAULT": "#ba1a1a",
        "0": "#000000",
        "10": "#410002",
        "20": "#690005",
        "25": "#7e0007",
        "30": "#93000a",
        "35": "#a80710",
        "40": "#ba1a1a",
        "50": "#de3730",
        "60": "#ff5449",
        "70": "#ff897d",
        "80": "#ffb4ab",
        "90": "#ffdad6",
        "95": "#ffedea",
        "98": "#fff8f7",
        "99": "#fffbff",
        "100": "#ffffff",
    },

```

**Comment by L-Blondy** [comment]
Fixed in 1.2.1

**Comment by sebastien-f** [comment]
Thanks for the fix ! I'll update ASAP. 
(and in case that was not planned, you left a [console.log](https://github.com/L-Blondy/tw-colors/blob/946798a80097412f1e8dd43f3a0998669066caeb/lib/index.ts#L104))
